.vonage/catalog-info.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: backstage.io/v1alpha1
+kind: System
+metadata:
+  name: video-network-test
+  title: Video Network Test
+  description: Library to check network connectivity to resources and services required to use Vonage Video API
+  annotations:
+    vonage.com/jira-id: VPF
+spec:
+  type: library
+  owner: group:orgdata/vcl
+  domain: video-api
+
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: video-network-test-android-ios
```

Comment:
The first component is the same name of the system `video-network-test` instead of `video-network-test-android-ios`.
You can have individual components one for android, one for ios, later on as a mono-repo.